frontend/src/lib/k8s/cluster.ts

```diff
@@ -111,6 +111,10 @@ makeKubeObject<T extends (KubeObjectInterface | KubeEvent)>(objectName: string):
       return this.jsonData![prop];
     }
 
+    getPods() {
+      return this.jsonData!.status.currentNumberScheduled;
+    }
+
```

Comment:
I think this shouldn't belong to the base class, because not all subclasses have this in their jsonData.
In the K8s API reference I see this is only in the DaemonSet spec: https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.20/#daemonset-v1-apps